turbo/jsonrpc/eth_call.go

```diff
@@ -574,6 +623,19 @@ func (api *BaseAPI) getWitness(ctx context.Context, db kv.RoDB, blockNrOrHash rp
 	touchedPlainKeys, touchedHashedKeys := store.Tds.GetTouchedPlainKeys()
 	codeReads := store.Tds.BuildCodeTouches()
 
+	// Verify that consistency of account and storage data for touched keys between the state reader
+	// and the patricia context (hph.ctx)
+	// If there is any inconsistency there will inevitably be a root hash mismatch later on when
+	// we try to construct the witness trie
+	isStateConsistent, err := verifyStateConsistency(touchedPlainKeys, store.Tds.StateReader, hph)
```

Comment:
id suppose to make code block like 
```go 
{
 // Verify that ....
 ...
}
```
so it could catch an eye that this section actually is for debugging and may be omitted.